stories/s5pl2.stories.js

```diff
@@ -0,0 +1,320 @@
+import {
+  S5PL2Layer,
+  setAuthToken,
+  isAuthTokenSet,
+  requestAuthToken,
+  CRS_EPSG3857,
+  CRS_EPSG4326,
+  BBox,
+  MimeTypes,
+  ApiType,
+  Polarization,
+  AcquisitionMode,
+  Resolution,
+} from '../dist/sentinelHub.esm';
+
+if (!process.env.INSTANCE_ID) {
+  throw new Error("INSTANCE_ID environment variable is not defined!");
+};
+
+if (!process.env.S5PL2_LAYER_ID) {
+  throw new Error("S5PL2_LAYER_ID environment variable is not defined!");
+};
+
+const instanceId = process.env.INSTANCE_ID;
+const layerId = process.env.S5PL2_LAYER_ID;
+
+export default {
+  title: 'Sentinel 5P L2',
+};
+
+export const getMapURL = () => {
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>GetMapUrl (WMS)</h2>";
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const layer = new S5PL2Layer(instanceId, layerId);
+
+  const bbox = new BBox(CRS_EPSG3857, -2035059.4, 15497760.4, -1956787.9, 15576031.8);
```

Comment:
Perhaps use some other bbox, I think these are out of s5 reach.

Reply:
I am not sure about that, but it's probably better to have the same bounding box for all the stories anyway - fixed.

Reply:
There is still a problem with switched X and Y in AbstractSentinelHubV3Layer.fetchTiles. As a consequence findTiles doesn't return correct results.

Reply:
Issue with switched coordinates will be resolved in some other branch.